src/Processor/Shortcode/assets/code-groups.css

```
@@ -0,0 +1,38 @@
+    padding: .55rem .8rem;
+    border: 0;
+    border-bottom: 2px solid transparent;
+    background: transparent;
+    color: inherit;
+    font: inherit;
+    white-space: nowrap;
+    cursor: pointer;
+}
+
+.code-group-tabs [role="tab"][aria-selected="true"] {
+    border-bottom-color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)